schema_based_on_diagram.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE patients{
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(50),
+    date_of_birth DATE NOT NULL
+}
+
+CREATE TABLE medical_histories{
+    id SERIAL PRIMARY KEY,
+     admitted_at TIMESTAMP NOT NULL,
+    patient_id INT NOT NULL,
+    status VARCHAR(50),
+    CONSTRAINT patients_fk FOREIGN KEY (patient_id) REFERENCES patients(id)
+}
+
+CREATE TABLE invoices{
+    id SERIAL PRIMARY KEY,
+    total_amount DECIMAL NOT NULL,
+    generated_at TIMESTAMP NOT NULL,
+    payed_at TIMESTAMP NOT NULL,
+    medical_history_id SERIAL NOT NULL,
+    CONSTRAINT history_fk FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
+}
+
+CREATE TABLE treatments{
+    id SERIAL PRIMARY KEY,
+    type VARCHAR(50),
+    name VARCHAR(50)
+}
+
+CREATE TABLE invoice_items(
+    id SERIAL PRIMARY KEY,
+    unit_price DECIMAL NOT NULL,
+    quantity INT NOT NULL,
+    total_price DECIMAL NOT NULL,
+    invoice_id SERIAL NOT NULL,
+    treatment_id SERIAL NOT NULL,
```

Comment:
- [ ] Let's replace `invoice_id SERIAL NOT NULL,` with `invoice_id INT NOT NULL,`
 - [ ] Let's replace `treatment_id SERIAL NOT NULL,` with `treatment_id INT NOT NULL,`